### DISCUSSION TITLE
how to hide osd-status-msg completely

### DISCUSSION THREAD

**Discussion by pjhfggij** [mpv-player/mpv#15261] [Q&A] [answered]
adding 
```
osd-status-msg
```
to ~.config/mpv/mpv.conf put empty string where osd-status-msg appears, which by default on archlinux is top left corner. However it still keeps small play icon - is there instead a way to get rid of it altogether (when pressing P or o)

**Comment by guidocella** [comment, marked as answer]
`--osd-msg3=' '`

**Reply by pjhfggij** [reply]
Thank you very much again, but it didn't work this time, I've added to ~/.config/mpv/mpv.conf  :
```
osd-msg1
osd-msg2
osd-msg3
```
but when playing mpv and pressing o for `o cycle-values osd-duration 3600000 100; show-progress
` this is what top left corner looks like:
![image](https://github.com/user-attachments/assets/2d17cb00-60e3-4610-aa3d-3116b2c33916)
I expected to nothing appear there. Adding `osd-status-msg` to mpv.conf at least gets rid of the time though: 
![image](https://github.com/user-attachments/assets/6b249c5b-b05e-4bc9-af9e-8c9fef0afc85)

**Comment by guidocella** [comment]
You didn't add `' '`.

**Reply by pjhfggij** [reply]
sorry, sorry you are right, it's working now 😅
